### ISSUE TITLE
Incorrect ' Missing trailing comma                        comma-dangle' for a variable with '()' that spans across multiple lines

### ISSUE THREAD

**Issue by kongakong** [eslint/eslint#8513]
**Tell us about your environment**

OSX 10.12.4

* **ESLint Version:** 3.19.0
* **Node Version:** v5.11.1
* **npm Version:** 3.10.6

Other eslint related in package.json
```
    "babel-eslint": "7.2.1",
    "eslint": "3.19.0",
    "eslint-config-airbnb": "14.1.0",
    "eslint-plugin-import": "2.2.0",
    "eslint-plugin-jsx-a11y": "4.0.0",
    "eslint-plugin-react": "6.10.3",
```

**What parser (default, Babel-ESLint, etc.) are you using?**

Babel-ESLint

**Please show your full configuration:**

```
{
  "extends": "airbnb",
  "env": {
    "browser": true,
    "mocha": true,
    "node": true
  },
  "globals": {
    "expect": true,
    "URLSearchParams": true
  },
  "parser": "babel-eslint",
  "rules": {
    "max-len": ["error", 80, 4, {"ignoreComments": true}],
    "react/jsx-filename-extension": "off",
    # Next 3 rules disabled for now until we decided what to do with
    # imports for tests
    "import/no-extraneous-dependencies": 0,
    "import/no-unresolved": 0,
    "import/extensions": 0
  },
  "plugins": [
    "react"
  ]
}




```

**What did you do? Please include the actual source code causing the issue.**

Flagged  `Missing trailing comma  comma-dangle` where it shouldn't be

```js


import { connect } from 'react-redux';
import Business from './Business';
import {
  fetchBusiness,
} from '../ducks/business';

const mapStateToProps = () => ({});
const mapDispatchToProps = dispatch => ({
  onSubmit: () => dispatch(fetchBusiness()),
});
export default connect(mapStateToProps, mapDispatchToProps)(
  Business
);



```

**What did you expect to happen?**

This line is good:
```
export default connect(mapStateToProps, mapDispatchToProps)(
  Business
);
```

But eslint wants me to add `,` after `Business`

**What actually happened? Please include the actual, raw output from ESLint.**

  12:11  error  Missing trailing comma  comma-dangle

**Comment by not-an-aardvark** [comment]
This is working as intended, see https://github.com/eslint/eslint/issues/7851
